applications/nrf_desktop/src/util/config_channel_transport.c

```diff
@@ -240,6 +240,25 @@ int config_channel_transport_set(struct config_channel_transport *transport,
 	return 0;
 }
 
+int config_channel_transport_get_disabled(uint8_t *buffer, size_t length)
+{
+	int err = frame_length_check(length);
+
+	if (!err) {
+		err = data_len_check(0);
```

Comment:
Isn't Build Assert already  checking this?

Reply:
It is. Thank you